exampleCourse/questions/element/sketch/question.html

```diff
@@ -0,0 +1,5 @@
+<pl-question-panel>
+  <p>Example use of a pl-sketch.</p>
+</pl-question-panel>
+
+<pl-sketch width=500 height=600></pl-sketch>
```

Comment:
Maybe make this wider? At least center it.

Reply:
For now, I made it wider... I wish there was a reasonable way to span the entire question panel without causing downstream issues (students on different device resolutions submitting things that look entirely different.)